econml/utilities.py

```diff
@@ -697,3 +698,84 @@ def predict(self, Xt):
         t = Xt[:, -self.n_T:]
         predictions = [self.model_list[np.nonzero(t[i])[0][0]].predict(X[[i]]) for i in range(len(X))]
         return np.concatenate(predictions)
+
+
+class StatsModelsWrapper:
+    """
+    Helper class to wrap a StatsModels OLS model to conform to the sklearn API.
+
+    Attributes
+    ----------
+    fit_args: dict of str: object
+        The arguments to pass to the `OLS` regression's `fit` method.  See the
+        statsmodels documentation for more information.
+    results: RegressionResults
+        After `fit` has been called, this attribute will store the regression results.
+    """
+
+    def __init__(self):
+        self.fit_args = {}
+
+    def fit(self, X, y, sample_weight=None):
+        """
+        Fit the ordinary least squares model.
+
+        Parameters
+        ----------
+        X : array-like, shape (n_samples, n_features)
+            Training data
+        y : array_like, shape (n_samples, 1) or (n_samples,)
+            Target values
+        sample_weight : array_like, shape (n_samples,)
+            Individual weights for each sample
+
+        Returns
+        -------
+        self
+        """
+        assert ndim(y) == 1 or (ndim(y) == 2 and shape(y)[1] == 1)
+        y = reshape(y, (-1,))
+        if sample_weight is not None:
+            ols = OLS(y, X, weights=sample_weight)
+        else:
+            ols = OLS(y, X)
```

Comment:
Check the statsmodel implementation I have in the dml proto. We might want to also support fit_intercept if we are building a generic wrapper. Though not needed for the dml application I guess

Reply:
I added this in the latest commit.